kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -364,6 +374,41 @@ object KyuubiConf {
       .version("1.4.0")
       .fallbackConf(FRONTEND_MAX_MESSAGE_SIZE)
 
+  val FRONTEND_THRIFT_HTTP_REQUEST_HEADER_SIZE: ConfigEntry[Int] =
+    buildConf("frontend.thrift.http.request.header.size")
+      .doc("Request header size in bytes, when using HTTP transport mode. Jetty defaults used.")
+      .version("1.5.0")
+      .intConf
+      .createWithDefault(6 * 1024)
+
+  val FRONTEND_THRIFT_HTTP_RESPONSE_HEADER_SIZE: ConfigEntry[Int] =
+    buildConf("frontend.thrift.http.response.header.size")
+      .doc("Response header size in bytes, when using HTTP transport mode. Jetty defaults used.")
+      .version("1.5.0")
+      .intConf
+      .createWithDefault(6 * 1024)
+
+  val FRONTEND_THRIFT_HTTP_MAX_IDLE_TIME: ConfigEntry[String] =
+    buildConf("frontend.thrift.http.max.idle.time")
+      .doc("Maximum idle time for a connection on the server when in HTTP mode.")
+      .version("1.5.0")
+      .stringConf
```

Comment:
timeConf

Reply:
@yaooqinn, this should be `stringConf`. I have handled the value of `frontend.thrift.http.max.idle.time` in `THttpFrontendService`.

Reply:
we don't align with any configuration with HiveConf, we shall keep our time/interval configurations behave the same.